lib/pester.rb

```diff
@@ -71,6 +71,7 @@ def self.retry_action(opts = {}, &block)
         end
       end
     end
+    nil
```

Comment:
You mentioned it before, if the post-retry exhaustion action doesn't raise, it used to return the number of retries it was supposed to make, which was dumb. when i was writing #7, I realized that in the trivial case of having it try 0 or "fewer" times, it would just do nothing and return that number. `nil` seems like it makes more sense. this should only be returned if max_attempts is < 1 or when `on_max_attempts_exceeded` is called with something that doesn't reraise.